rotkehlchen/tests/unit/test_usage_analytics.py

```diff
@@ -4,16 +4,15 @@
 from rotkehlchen.utils.misc import get_system_spec
 
 
-def test_create_usage_analytics():
-    analytics = create_usage_analytics()
+def test_create_usage_analytics(data_dir):
+    analytics = create_usage_analytics(data_dir)
 
     assert 'system_os' in analytics
     assert 'system_release' in analytics
     assert 'system_version' in analytics
     assert analytics['rotki_version'] == get_system_spec()['rotkehlchen']
     if sys.platform != 'darwin':
-        assert analytics['country'] != 'unknown'
-        assert analytics['city'] != 'unknown'
+        assert analytics['city'] == 'unknown'
```

Comment:
Had to remove this from tests since it fails to detect the ip in tests

Reply:
understandable. We would need to test it in few different networks to make sure it works -- by hand.